doc/developing.md

```diff
@@ -97,4 +97,23 @@ When updating the supported GHC versions, remember to update:
 - [The README](../README.md)
 - [The Cabal file](../llvm-pretty-bc-parser.cabal)'s `Tested-with` field
 - [The Nix flake](../flake.nix)
+
+  Run `nix flake update` (installing [`nix`](https://nixos.org) first if
+  necessary) which will update the `flake.lock` file in the top level directory.
+  Optionally run `nix build ./#TESTS` to run the nix-based tests locally.  Then
+  commit the updated `flake.lock` file.
+
+  Newer GHC versions are automatically available to the nix build, but may
+  require updating the flake lockfile as described above to become visible.  To
+  add new GHC versions to the `nix` builds, see the `ghc-version` list in the
+  `nix-ci.yml` file.
```

Comment:
Minor point, but to me it seems like this last sentence (starting with "To") is what developers should try first, and the rest is details on what to do if that doesn't work. Perhaps this sentence could be the first part of this section?